website/static/js/osfHelpers.js

```diff
@@ -744,6 +752,50 @@ var any = function(listOfBools) {
     return false;
 };
 
+/** 
+ * A helper for creating a style-guide conformant bootbox modal. Returns a promise.
+ * @param {String} title: 
+ * @param {String} message:
+ * @param {String} actionButtonLabel:
+ * @param {Object} options: optional options
+ * @param {String} options.actionButtonClass: CSS class for action button, default 'btn-success'
+ * @param {String} options.cancelButtonLabel: label for cancel button, default 'Cancel'
+ * @param {String} options.cancelButtonClass: CSS class for cancel button, default 'btn-default'
+ *
+ * @example
+ * dialog('Hello', 'Just saying hello', 'Say hi').done(successCallback).fail(doNothing);
+ **/
+var dialog = function(title, message, actionButtonLabel, options) {
```

Comment:
It is in the admin app. If need be I can remove it, but it'll just get added later.